pkg/virtctl/portforward/portforward.go

```diff
@@ -78,19 +77,23 @@ func NewCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
 }
 
 type PortForward struct {
-	address      *net.IPAddr
-	clientConfig clientcmd.ClientConfig
```

Comment:
nit: instead of removing the clientConfig can we replace this with the command. So, in this way we avoid to pass the `clientConfig` to the functions